visualization/app/codeCharta/core/settings/settings.service.ts

```diff
@@ -85,19 +85,19 @@ export class SettingsService implements DataServiceSubscriber, CameraChangeSubsc
         });
     }
 
-    private getInitialSettings(renderMap: any, metrics: string[]): Settings {
+    public getDefaultSettings(renderMap: any, metrics: string[]): Settings {
```

Comment:
it propably was like this before but is it possible to type the map correctly ?

Reply:
Good point, I'll check it.